cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -145,6 +145,14 @@ func VerifyBlobCmd(ctx context.Context, ko options.KeyOpts, certRef, certEmail,
 			return err
 		}
 		if certChain == "" {
+			co.RootCerts, err = fulcio.GetRoots()
```

Comment:
I could move this to a blanket `if certChain == ""` outside the switch.

We would then load it in even if we are using a pubkey and it's not needed.

Reply:
But that would add a network call where none is needed; I think we want to avoid that

Reply:
I think this is fine as-is, it's clearer to load in the chain around the point of verification imo - This probably needs a larger rewrite anyways.

Reply:
Yeah, I kept this as is to avoid the network call, sounds good. let's keep as is. Just wanted to keep code simple.